lib/workers/miq_defaults.rb

```diff
@@ -0,0 +1,26 @@
+require "active_support/core_ext/numeric/time"
+
+module Workers
+  class MiqDefaults
+    HEARTBEAT_TIMEOUT = 2.minutes.freeze
+    STARTING_TIMEOUT  = 10.minutes.freeze
+    STOPPING_TIMEOUT  = 10.minutes.freeze
```

Comment:
These were removing hardcoded values that existed in the code base of just `10.minutes` (see the diff).  There wasn't any reference to Settings with what I replaced these with.  If these shouldn't be hardcoded in the first place, that is an entirely different question that I don't have the answer to.

As far as why it is it's own class, since we want to have the `heartbeat_check` execute as simply as possibly, without needing external dependencies, so making it it's own class allows it to be reused in both the codebase and in this script.